src/services/shopping-cart.service.ts

```diff
@@ -7,8 +8,17 @@ import { Product } from 'src/interfaces/productInterface';
 export class ShoppingCartService {
 
   //constructor() { }
+  products: Product[] = [];
 
-  onAddToCartClicked(productName: string){
-    alert('You have added ' + productName + ' To your cart!');
+   onAddToCartClicked(product: Product){
```

Comment:
Sounds more like an event listener name than an action. We could probably name this something more general like `addProductToCart`.  
```suggestion
   addProductToCart(product: Product){
```